Carpini/update dapp kit doc v1

davidecarpini opened the pull request.

nwbrettski commented on core-concepts/evm-compatibility/test-coverage/README.md:
Three links are not working.

nwbrettski commented on core-concepts/evm-compatibility/test-coverage/eth_sign.md:
Link isn't working.

nwbrettski commented on developer-resources/getting-started.md:
Is the link here correct. Going to a v0 page of the dapp-kit?

nwbrettski commented on developer-resources/sdks-and-providers/dapp-kit/dapp-kit-1/v0-to-v1.md:
Would you consider having a changelog in the future to track changes between versions instead of having a page per version change. Just a suggestion to help scale this approach.

nwbrettski commented on start-building/tutorials/account-abstraction/configure-and-run-the-trampoline-wallet.md:
Link isn't working.

nwbrettski reviewed (changes requested):
A couple of link updates required, not sure if these broken links are a result of GitBook syncing or not.

davidecarpini commented:
thanks @nwbrettski , I reverted all the changes of the gitbook sync (all the link errors).
About the versioning strategy, this is probably the most trivial one, it's just a copy and paste changing new stuff. I added a v0 to v1 changelog, but I would leave the v0 folder for now. I didn't find a better and clearer way for the versioning.
Aave uses a menu to change the SDK version, but it applies to the entire doc.
https://docs.aave.com/developers/v/2.0/getting-started/sdks
<img width="1728" alt="Screenshot 2024-01-15 at 09 51 24" src="https://github.com/vechainfoundation/vechain-docs/assets/5785838/26336840-3473-4057-961c-5043473899de">

darrenvechain commented:
> thanks @nwbrettski , I reverted all the changes of the gitbook sync (all the link errors). About the versioning strategy, this is probably the most trivial one, it's just a copy and paste changing new stuff. I added a v0 to v1 changelog, but I would leave the v0 folder for now. I didn't find a better and clearer way for the versioning. Aave uses a menu to change the SDK version, but it applies to the entire doc. https://docs.aave.com/developers/v/2.0/getting-started/sdks <img alt="Screenshot 2024-01-15 at 09 51 24" width="1728" src="https://private-user-images.githubusercontent.com/5785838/296702460-26336840-3473-4057-961c-5043473899de.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUzMTU0NzIsIm5iZiI6MTcwNTMxNTE3MiwicGF0aCI6Ii81Nzg1ODM4LzI5NjcwMjQ2MC0yNjMzNjg0MC0zNDczLTQwNTctOTYxYy01MDQzNDczODk5ZGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDExNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMTVUMTAzOTMyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OWU3NTA3ZmMzYzUyM2EwMmY5ZDhkZjlmMTBiN2NlODQyNzZkMTEzYzY5MjVjNDIwNGYwZmQ1MDBlZDZhYzYxZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Oe0cP4VcMPrc1WMCK5aKQYg2xMDvSNJLo8h-xIjsYYk">

@davidecarpini IMO that example is kind of different as it looks like they're exposing the 2 different versions in the same SDK. I personally think its better to prune any existence of v0

nwbrettski reviewed (approved):
Thanks for the updates.
